GPU/D3D11/D3D11Util.cpp

```diff
@@ -161,15 +163,15 @@ void StockObjectsD3D11::Create(ID3D11Device *device) {
 
 void StockObjectsD3D11::Destroy() {
 	for (int i = 0; i < 16; i++) {
-		blendStateDisabledWithColorMask[i]->Release();
+		blendStateDisabledWithColorMask[i] = nullptr;
 	}
-	depthStencilDisabled->Release();
-	depthDisabledStencilWrite->Release();
-	rasterStateNoCull->Release();
-	samplerPoint2DWrap->Release();
-	samplerLinear2DWrap->Release();
-	samplerPoint2DClamp->Release();
-	samplerLinear2DClamp->Release();
+	depthStencilDisabled = nullptr;
```

Comment:
this may not be unsafe, but looks scary to me. How about using ->Reset() ?

Reply:
I could, but do you want me to replace `= nullptr` with `.Reset()` everywhere or here only?

Reply:
Everywhere preferably

Reply:
I've done it in D3D11 code.